chromium_src/chrome/browser/prefs/browser_prefs.cc

```diff
@@ -21,8 +21,8 @@
 #undef MigrateObsoleteProfilePrefs
 
 #if BUILDFLAG(BRAVE_WALLET_ENABLED)
-#include "brave/common/extensions/extension_constants.h"
 #include "extensions/browser/extension_prefs.h"
+#include "extensions/common/constants.h"
```

Comment:
this doesn't appear to be used, same with extension_prefs

Reply:
Addressed by 97c915735c35d17a919dfa8c0a26a0f238d431fe